integration-tests/tests/src/setup-globals.ts

```diff
@@ -54,6 +54,10 @@ chai.use(chaiRealmObjects);
 const DEFAULT_LONG_TIMEOUT = 30 * 1000; // 30s
 
 describe("Test Harness", function (this: Mocha.Suite) {
+  // Inject the root suite title from the global
+  if (typeof title === "string" && this.parent?.root) {
+    this.parent.title = title;
+  }
```

Comment:
This was needed to get the title from "inject-dev-environment.ts" to propagate as expected.